mods/d2k/maps/atreides-04/atreides04-AI.lua

```diff
@@ -36,7 +36,7 @@ SendAttack = function(owner, size)
 	Utils.Do(units, IdleHunt)
 
 	if #units > 0 then
-		Media.DisplayMessage("Harkonnen units approaching!", "Fremen Leader")
+		Media.DisplayMessage(UserInterface.Translate("harkonnen-units-approaching"), UserInterface.Translate("fremen-leader"))
```

Comment:
it doesn't looks like SendAttack is even called anywhere

Reply:
ah, it's supposed to be called by ProduceUnits(), but I can't get this notification ingame